web/src/actions/apd.js

```diff
@@ -276,9 +276,20 @@ export const submitAPD = (save = saveApd) => (dispatch, getState) =>
     .then(() => {
       dispatch({ type: SUBMIT_APD_REQUEST });
 
-      const { apd: { data: { id: apdID } }, budget } = getState();
+      const {
+        activities: { byKey: activities },
+        apd: { data: { id: apdID } },
+        budget
+      } = getState();
 
       const tables = {
+        activityQuarterlyFederalShare: Object.entries(budget.activities).reduce(
+          (acc, [key, activity]) => ({
+            ...acc,
+            [activities[key].name]: activity.quarterlyFFP
```

Comment:
Seemed reasonable to key off the activity name instead of activity ID, since activity names are unique within an APD.  More human readable this way, and pretty minimal effort for a machine to figure out the link too.